unittests/Sema/SemaFixture.h

```diff
@@ -0,0 +1,65 @@
+//===--- SemaFixture.h - Helper for setting up Sema context -----*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/AST/ASTContext.h"
+#include "swift/AST/DiagnosticEngine.h"
+#include "swift/AST/Module.h"
+#include "swift/AST/SourceFile.h"
+#include "swift/AST/Type.h"
+#include "swift/Basic/LangOptions.h"
+#include "swift/Basic/Platform.h"
+#include "swift/Basic/SourceManager.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Host.h"
+#include "gtest/gtest.h"
+#include <string>
+
+namespace swift {
+namespace unittest {
+
+class SemaTestBase : public ::testing::Test {
+public:
+  LangOptions LangOpts;
+  TypeCheckerOptions TypeCheckerOpts;
+  SearchPathOptions SearchPathOpts;
+  ClangImporterOptions ClangImporterOpts;
+  SourceManager SourceMgr;
+  DiagnosticEngine Diags;
+
+  SemaTestBase() : Diags(SourceMgr) {
+    LangOpts.Target = llvm::Triple(llvm::sys::getProcessTriple());
+
+    std::string libDir = SWIFTLIB_DIR;
+    std::string platform = getPlatformNameForTriple(LangOpts.Target).str();
+
+    SearchPathOpts.RuntimeResourcePath = libDir;
+    SearchPathOpts.RuntimeLibraryPaths.push_back(libDir + "/" + platform);
```

Comment:
This path separator will cause you trouble on Windows. LLVM must have a library for this somewhere, I am sure...

Reply:
I'll take a look!

Reply:
Fixed!